Library/Homebrew/dependency_collector.rb

```diff
@@ -161,7 +161,7 @@ def resource_dep(spec, tags)
     elsif strategy <= FossilDownloadStrategy
       Dependency.new("fossil", tags)
     elsif strategy <= BazaarDownloadStrategy
```

Comment:
Should we rename this?

Reply:
I think the repository format is still called "Bazaar", so keeping the name is probably still appropriate.

From the Breezy [README](https://github.com/breezy-team/breezy/blob/149cab0ce042ff956f085ea60218ad533d6e1bab/README.rst):

> By default, Breezy provides support for both the [Bazaar](https://www.bazaar-vcs.org/) and [Git](https://www.git-scm.com/) file formats.

Reply:
Cool, thanks, fine with me then 👍🏻